doc/classes/Expression.xml

```diff
@@ -19,18 +37,21 @@
 			<argument index="2" name="show_error" type="bool" default="true">
 			</argument>
 			<description>
+				Executes the expression that was previously parsed by [method parse] and returns the result. Before you use the returned object, you should check if the method failed by calling [method has_execute_failed].
 			</description>
 		</method>
 		<method name="get_error_text" qualifiers="const">
 			<return type="String">
 			</return>
 			<description>
+				Returns the error text if [method parse] has failed.
 			</description>
 		</method>
 		<method name="has_execute_failed" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if [method execute] has failed.
```

Comment:
Our convention is to use code markup for `true` and `false` when used as boolean values, so it should be `[code]true[/code]`.